services/todo-ui/src/server/.generated/proxies/rest/Tasks-routes.ts

```diff
@@ -0,0 +1,16 @@
+//
+// GENERATED SOURCE - DO NOT EDIT
+//
+import { Router } from "express";
```

Comment:
Looks like we don't use our prettier config when we generate code. We have `singleQuote: true` https://github.com/kapetacom/eslint-prettier-configs/blob/master/packages/prettier-config/prettier-config.json#L4C6-L4C17

Reply:
Add ticket (this is already a very big change )